umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/monitoring.md

```diff
@@ -43,18 +43,21 @@ deploy a data source.
 After logging in, the Instance Dashboard page shows the **Overview** tab.
 You can click and open each status on this tab.
 
-![Instance Overview](/images/features/monitor-management/instanceOverview.png?width=80%)
+![Instance Overview](/images/features/monitor-management/instanceOverviewDemo.png?width=80%)
 
 The **Connection Management** tab shows the status of all the instance's connections and their associated
 data sources. Moreover, you can create a new connection, as well as initialize them.
 Read more about the Connection Management in the [Connectivity](/docs/features/connectivity/) section.
 
-![Connection Management](/images/features/monitor-management/instanceConnectionManagement.png?width=80%)
+![Connection Management](/images/features/monitor-management/connectionManagementDemo.png?width=80%)
 
-The Data Dashboard displays the topic structure and publisher & subscriber
-information.
+<!-- TODO: Mention the payload's values history feature once implemented -->
 
-![Data Dashboard](/images/features/monitor-management/dataDashboard.png?width=80%)
+The **Tag Browser** displays the topic structure and publisher & subscriber
```

Comment:
It does not display the topic structure! It displays the tag structure. Best is to give an example and explain it:

1. A tag is typically a "variable" which belongs to a device somewhere in a ISA-95 structure
2. There are multiple technical options to transport the tag across a UNS and into the database. You could create a folder structure inside of the topic structure, or you could work with JSON objects inside of the payload (link to documentation). You can send every "tag" into the same topic or use different sub-topics. Then because of how MQTT and Kafka work, the topics will be different. To summarize there: MQTT Topic = Kafka topic + Kafka Key (link to merge point)

The tag browser allows it now an automation engineer to browse all his tags iwthout thinking about the technical backgrounds like topics, keys, payload structure etc.

(in the future, there will be more information about the IT origin of the tag as well like the kafka topic, key, mqtt topic, etc.)